Exiled.API/Features/Cassie.cs

```diff
@@ -100,11 +103,11 @@ public static void SCPTermination(Player scp, DamageHandlerBase info)
         /// <summary>
         /// Announce the termination of a custom SCP name.
         /// </summary>
-        /// <param name="scpname">SCP Name.</param>
+        /// <param name="scpName">SCP Name.</param>
         /// <param name="info">Hit Information.</param>
-        public static void CustomSCPTermination(string scpname, DamageHandlerBase info)
+        public static void CustomSCPTermination(string scpName, DamageHandlerBase info)
```

Comment:
I'd explain how the "SCP Name" should be formatted, because cassie won't say "scp457", but for "scp 4 5 7" is different.
Also, DamageHandlerBase is pretty meh. Use Exiled's DamageHandler.
I don't really understand the "result" assignment.
I'd rename the method to _CustomScpTermination_.

Reply:
I didn't add the method, I just renamed the parameter. I'll look at it though

Reply:
Result is assigned because CASSIE uses it at the end of the method

Reply:
Oh yeah, I was from cell, I didn't see that clearly 👍